src/box/alter.cc

```diff
@@ -2500,6 +2501,10 @@ on_replace_dd_space(struct trigger * /* trigger */, void *event)
 		}
 		alter_guard.is_active = false;
 	}
+	/*
+	 * After cache modification transaction is now allowed to yield.
+	 */
+	memtx_tx_acquire_ddl(txn);
```

Comment:
Having this call in each DDL trigger function is fragile. It'd be nicer to have it in some sort of trigger. We used to have on_stmt_begin trigger, which I removed a while back (e5c4ce75f86d279e6a044adffa3fb99cced0ded4). Worth resurrecting it?